rancher/v1.5/en/kubernetes/upgrading/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Upgading Kubernetes
+layout: rancher-default-v1.5
+version: v1.5
+lang: en
+---
+
+# Upgrading Kubernetes
+---
```

Comment:
I wanted to get this merged out first and then i'll add in details from your ticket. :)